www/class/centreon-knowledge/wikiApi.class.php

```diff
@@ -72,10 +72,12 @@ public function __construct()
     private function getCurl()
     {
         $curl = curl_init();
-
+        $cookiefile = tempnam("/tmp", "CURLCOOKIE");
         curl_setopt($curl, CURLOPT_URL, $this->url);
         curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
         curl_setopt($curl, CURLOPT_POST, true);
+        curl_setopt($curl, CURLOPT_COOKIEFILE, $cookiefile);
```

Comment:
can you add an expiration time using for example `CURLOPT_CONNECTTIMEOUT`

Reply:
added removal of cookies as agreed

Reply:
thanks